packages/path_provider/path_provider/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: path_provider
 description: Flutter plugin for getting commonly used locations on host platform file systems, such as the temp and app data directories.
 homepage: https://github.com/flutter/plugins/tree/master/packages/path_provider/path_provider
-version: 2.0.0-nullsafety
+version: 2.0.1-nullsafety
```

Comment:
The version should be bumped in conjunction with updating the minimum required version of `path_provider_platform_interface` (and that'll need to be a second PR, after the latter is published). Currently you're changing the version of this package even though nothing has actually changed.

And conversely, you're changing `path_provider_platform_interface` without updating its version, which means the change can't be published.

Reply:
I now changed the version as suggested in both files and updated the changelog accordingly. Please review again.